ci/deploy.sh

```diff
@@ -35,15 +35,9 @@ set +o xtrace  # disable xtrace in subshell for private key operations
 if [ -v MANUBOT_SSH_PRIVATE_KEY ]; then
   base64 --decode <<< "$MANUBOT_SSH_PRIVATE_KEY" | ssh-add -
 else
-echo "DeprecationWarning: Loading deploy.key from an encrypted file.
-In the future, using the MANUBOT_SSH_PRIVATE_KEY environment variable may be required."
-openssl aes-256-cbc \
-  -K $encrypted_9befd6eddffe_key \
-  -iv $encrypted_9befd6eddffe_iv \
-  -in ci/deploy.key.enc \
-  -out ci/deploy.key -d
-chmod 600 ci/deploy.key
-ssh-add ci/deploy.key
+  echo >&2 "Skipping deployment because the MANUBOT_SSH_PRIVATE_KEY environment variable is not set.
+  Instructions at https://github.com/manubot/rootstock/blob/master/SETUP.md#deploy-key"
```

Comment:
I noticed these instructions still contain
> For "Title", add a description like "Manubot Travis Deploy Key".

which could be changed to "Manubot GitHub Deploy Key"